recipe/meta.yaml

```diff
@@ -1,35 +1,37 @@
+{% set name = "twine" %}
 {% set version = "3.7.1" %}
 
 package:
-  name: twine
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/t/twine/twine-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 28460a3db6b4532bde6a5db6755cf2dce6c5020bada8a641bb2c5c7a9b1f35b8
 
 build:
   noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
-  skip: true  # [py<36]
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  # depends on readme_renderer not available for linux-s390x
```

Comment:
Shouldn't we just rebuilt `readme_renderer` for s390x? I think it's a pure python package...

Reply:
That's cool, I've opened a follow-up ticket for `readme_renderer` https://anaconda.atlassian.net/browse/PKG-2328

This will wait.

Reply:
`readme_renderer` done by @JamesRobertsonGames